src/components/Shared/DeleteConfirmation.js

```diff
@@ -0,0 +1,77 @@
+import React, { useContext, useState } from 'react'
+import CustomTextField from '../Inputs/CustomTextField'
+import WindowToolbar from './WindowToolbar'
+
+const DeleteConfirmation = ({ window, props, obj }) => {
+  const [deleteConfirmation, setDeleteConfirmation] = useState('')
+  const [isDeleteDisabled, setIsDeleteDisabled] = useState(true)
+
+  const handleChange = event => {
+    const value = event.target.value
+    setDeleteConfirmation(value)
+    setIsDeleteDisabled(value.toLowerCase() !== 'delete')
+  }
+
+  const handleClear = () => {
+    setDeleteConfirmation('')
+    setIsDeleteDisabled(true)
+  }
+
+  const handleSubmit = async () => {
+    if (deleteConfirmation.toLowerCase() === 'delete') {
+      props.onDeleteConfirmation(obj)
+      window.close()
+    }
+  }
+
+  const actions = [
+    {
+      key: 'Delete',
+      condition: true,
+      onClick: handleSubmit,
+      disabled: isDeleteDisabled
+    }
+  ]
+
+  return (
+    <div
+      style={{
```

Comment:
not sure about the inline styling

Reply:
display: 'flex',
        flexDirection: 'column',
        height: '100vh',
        justifyContent: 'space-between',